sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suite.scala

```diff
@@ -384,4 +388,32 @@ class HiveThriftServer2Suite extends FunSuite with Logging {
       }
     }
   }
+
+  test("SPARK-5100 monitor page") {
```

Comment:
@JoshRosen, I have talked with @liancheng about the UISeleniumSuite. I did not add more complex web UI tests because that we worried about the test costs too much time.